From af4051c7c4cf933a9e5537e2729aeb4cd237a56c Mon Sep 17 00:00:00 2001
From: Potin Lai <potin.lai@quantatw.com>
Date: Mon, 18 Sep 2023 16:49:24 +0800
Subject: [PATCH 1/1] configurations: update Bletchley FSC configuration

Update FSC configuration to meet the requirements of thermal team.
Version: R08, FSC_A_1003

Signed-off-by: Potin Lai <potin.lai@quantatw.com>
Change-Id: I8ae8a61ad708d2d6e8f86adc44cde8cb207e8f01
---
 configurations/bletchley_chassis.json | 86 +++++----------------------
 1 file changed, 15 insertions(+), 71 deletions(-)

diff --git a/configurations/bletchley_chassis.json b/configurations/bletchley_chassis.json
index 69129db..253b032 100644
--- a/configurations/bletchley_chassis.json
+++ b/configurations/bletchley_chassis.json
@@ -220,24 +220,24 @@
                 "Virtual_Inlet_Temp"
             ],
             "Name": "inlet_temp",
-            "NegativeHysteresis": 1,
+            "NegativeHysteresis": 0,
             "Output": [
-                15.0,
-                15.0,
-                15.0,
-                15.0,
-                16.0,
-                17.0,
-                18.0,
-                19.0,
-                20.0,
-                21.0,
-                22.0,
-                23.0,
-                25.0,
                 27.0,
+                30.0,
+                34.0,
+                37.0,
+                40.0,
+                44.0,
+                45.0,
+                46.0,
                 47.0,
+                48.0,
+                49.0,
                 50.0,
+                50.0,
+                53.0,
+                55.0,
+                60.0,
                 80.0
             ],
             "PositiveHysteresis": 0,
@@ -312,7 +312,7 @@
             ],
             "PositiveHysteresis": 0,
             "Reading": [
-                39.0,
+                35.0,
                 40.0,
                 43.0,
                 44.0,
@@ -362,62 +362,6 @@
             "Zones": [
                 "Zone0"
             ]
-        },
-        {
-            "Class": "temp",
-            "Inputs": [
-                "nvme1",
-                "nvme2",
-                "nvme3",
-                "nvme4",
-                "nvme5",
-                "nvme6"
-            ],
-            "Name": "nvme_temp",
-            "NegativeHysteresis": 3,
-            "Output": [
-                0.0,
-                22.0,
-                22.0,
-                24.0,
-                28.0,
-                30.0,
-                32.0,
-                34.0,
-                35.0,
-                37.0,
-                39.0,
-                42.0,
-                43.0,
-                44.0,
-                45.0,
-                50.0,
-                80.0
-            ],
-            "PositiveHysteresis": 1,
-            "Reading": [
-                35.0,
-                38.0,
-                44.0,
-                46.0,
-                48.0,
-                49.0,
-                50.0,
-                51.0,
-                52.0,
-                54.0,
-                55.0,
-                56.0,
-                57.0,
-                58.0,
-                59.0,
-                64.0,
-                67.0
-            ],
-            "Type": "Stepwise",
-            "Zones": [
-                "Zone0"
-            ]
         }
     ],
     "Name": "Bletchley Chassis",
-- 
2.31.1

